test/robustness/convex_hull/random_multi_points.cpp

```diff
@@ -170,20 +165,21 @@ int main(int argc, char** argv)
 
         std::string type = "double";
         int count = 1;
-        int seed = static_cast<unsigned int>(std::time(0));
-        int pcount = 3;
+        int seed = -1;
+        int pcount = 30;
         int field_size = 10;
         settings_type settings;
 
         description.add_options()
             ("help", "Help message")
             ("seed", po::value<int>(&seed), "Initialization seed for random generator")
-            ("count", po::value<int>(&count)->default_value(1), "Number of tests")
```

Comment:
Is this not needed anymore? Seems unrelated to boost random replacement.

Reply:
hmm, thanks! I have to check this indeed. Looks like an unwanted omission.

Reply:
It's fine. Indeed not really unrelated to Boost.Random - but sidely because I changed some of these in touched code.
The `->default_value` is not necessary if the variable is already initialized like that. And in this case (and many more), it is.